action.sh

```diff
@@ -46,6 +52,15 @@ trap cleanup EXIT
 
 prepare () {
     pip3 install -r $ACTION_PATH/requirements.txt
+
+    # Overwrite roles file if it exists already
+    echo "#!/bin/bash" > $ROLES_FILE_PATH
+    echo "set -e" >> $ROLES_FILE_PATH
+
+    for ROLE in ${REQUIRED_ROLES}; do
+        echo echo "psql --username \"$DB_USER\" --dbname \"$DB_NAME\"  CREATE ROLE \"$ROLE\" LOGIN" >> $ROLES_FILE_PATH
```

Comment:
What does the double `echo` do? You also have a double space before `CREATE ROLE`

Reply:
It was a bug, nice spotting.